tooling/cli/package-lock.json

```diff
@@ -1,602 +0,0 @@
-{
```

Comment:
not sure if the package lock files under workspaces are intended to stay. remembered earlier the lock files were generated when using `lerna` to manage workspaces. Since we moved to `npm` to manage workspaces, we have a single lock file under root. might consider cleaning up these workspace lock files? (also ones from `tooling/helium-server`, `packages/contact-block`.)

Reply:
@ljiang-ti Yeah, think cleaning up the workspaces here makes sense since `learn` isn't being used to manage workspaces.